.github/workflows/diff-report.yml

```diff
@@ -97,6 +97,36 @@ jobs:
           # shellcheck disable=SC2005
           echo "$(cat "$GITHUB_OUTPUT")"
 
+      - name: Extract config name
+        id: extract_config_name
+        run: |
+          CONFIG_NAME=$(grep -oP '(?<=Diff Regression config: ).*?(?=/config.xml)' .ci-temp/text | awk -F'/' '{print $(NF-1)"/"$NF}')
+          echo "config_name=$CONFIG_NAME" >> $GITHUB_OUTPUT
+          echo "Extracted config name: $CONFIG_NAME"
+
+      - name: Extract check and example
+        id: extract_check_example
+        run: |
+          CHECK_EXAMPLE=$(echo "${{ github.event.comment.body }}" | grep -oP '(?<=Github, generate report ).*')
```

Comment:
we need trim spaces right in comment content.
We need to have trigger phrase to be `Github, generate report for `.
To get Check name , we need to change comment by replace of `Github, generate report for ` with ``.
all other content is path in test-config repo.